### ISSUE TITLE
[Bug] Evo 3.1.29 getCacheFolder

### ISSUE THREAD

**Issue by Nicola1971** [evocms-community/evolution#490]
### Summary

Evo 3.1.29 return just 'cache/' with $modx->getCacheFolder()

### Steps to reproduce.

Evo 1.4.35 with $modx->getCacheFolder()

 'assets/cache/'
 
 Evo 3.1.29 with $modx->getCacheFolder()
 
 'cache/' instead of 'core/storage/bootstrap'

### EvoCMS version

Evo 3.1.29

### PHP version

8.1.29

**Comment by Pathologic** [comment]
What's your case? Do you need to cache something?

**Comment by Nicola1971** [comment]
Just testing a Evo 1.4 plugin with Evo 3x.
Anyway, I just found this bug, reported by you, five years ago :) https://github.com/evolution-cms/evolution/issues/998

**Comment by Pathologic** [comment]
@Nicola1971 I've declared this method as deprecated, so it would be better not to use it at all. In the case of your PageCacheManager plugin, you needn't getCacheFolder, because you can call $modx->clearCache($doc_id) instead of all this code: https://github.com/Nicola1971/PageCacheManager/blob/main/install/assets/plugins/PageCacheManager.tpl#L64-L101;

**Comment by Nicola1971** [comment]
Is there anyway a replacement for getCacheFolder? without hardcoding the path?
I didn't know you could use clearcache with doc_id! I don't think I ever read that in the documentation. Thanks a lot!

**Comment by Nicola1971** [comment]
clearCache($doc_id) streamlined my code a lot :)

**Comment by Pathologic** [comment]
I don't think we need a replacement, because there's [Storage facade](https://laravel.com/docs/8.x/filesystem) in 3.x.

**Comment by Nicola1971** [comment]
is there still clearcache on 3x?

**Comment by Pathologic** [comment]
Yes.
